### PR TITLE
Refactor Supplementary Billing Flags

### PR DESCRIPTION
**Pull request by Beckyrose200**
https://eaflood.atlassian.net/browse/WATER-4137
https://eaflood.atlassian.net/browse/WATER-4199
https://eaflood.atlassian.net/browse/WATER-4622
https://eaflood.atlassian.net/browse/WATER-4588

Upon completion of the above two-part tariff supplementary tickets, it was noted that there are now multiple places in the code that handles flagging for pre-sroc supplementary, sroc supplementary and now two-part tariff supplementary. The UI has its own flagging mechanisms as well as the service repo and now our new code in system.

To keep things consistent, and easy to maintain it was decided that all the code for flagging a supplementary bill run be consolidated into the system repo. All other flagging mechanisms can be removed meaning there is only 1 place to update and maintain flagging. This PR is a housekeeping task to reduce the code.

### CODE REVIEW

File: src/lib/connectors/system/workflow-supplementary-billing.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const config = require('../../../../config')
+const { serviceRequest } = require('@envage/water-abstraction-helpers')
+
+/**
+ * Posts to system repo to flag new charge versions for SROC supplementary billing
```
Comment (Jozzey) [inline src/lib/connectors/system/workflow-supplementary-billing.js:7]: Not sure if this comment is correct or a copy/paste error from `src/lib/connectors/system/charge-version-supplementary-billing.js`?

**Review by Jozzey — COMMENTED** [review]
I'm not 100% sure if that comment is correct or not. If you can confirm one way or the other I'll approve this.

File: src/lib/connectors/system/workflow-supplementary-billing.js (hunk 1 above)
Comment (Beckyrose200) [reply src/lib/connectors/system/workflow-supplementary-billing.js]: Good spot, thank you

**Review by Jozzey — APPROVED** [review]

**Review by Jozzey — APPROVED** [review]
👍🏼

**Pull request merged**